mods/persistence/_persistence.dme

```diff
@@ -1,19 +1,19 @@
 #ifndef MODPACK_PERSISTENCE
```

Comment:
I suggest you revert your persistence.dme to before you restored it, because the diff is acting funky with all the lines being moved around, I had to run winmerge on it to make sure what was going on. 
Also, seems like all the files you added under "modules/client/" aren't included in the persistence.dme. They won't get compiled if you don't include them.

Reply:
frick